lib/compat/wordpress-6.8/preload.php

```diff
@@ -10,8 +10,16 @@
  */
 function gutenberg_block_editor_preload_paths_6_8( $paths, $context ) {
```

Comment:
I think ideally, we should build some kind of "router" that has equivalent paths to the router we use in the site editor. But this address the important use-case.

Reply:
Also would be cool to kind of allow using packages from packagist. Matching routes is a solved problem already in PHP in different packages.